packages/components/src/components/single-select/controller.ts

```diff
@@ -1,6 +1,5 @@
-import type { Option, OptionsPropType, SelectOption, SingleSelectProps, SingleSelectWatches, StencilUnknown, W3CInputValue } from '../../schema';
-import { watchValidator } from '../../schema';
-import { validateOptions, watchBoolean, watchString } from '../../schema';
+import { Option, OptionsPropType, SelectOption, SingleSelectProps, SingleSelectWatches, StencilUnknown, W3CInputValue, watchNumber } from '../../schema';
```

Comment:
`type` weg?!

Reply:
ich schaue es noch gleich und passe die Kommentaren an.

Reply:
@donchi-donald - alles gut, ich fix das fix. Dann kann es in den Develop bzw. Release-2